companies/volkswagen-ag.json

```diff
@@ -0,0 +1,21 @@
+{
+    "slug": "volkswagen-ag",
+    "relevant-countries": [
+        "all"
+    ],
+    "categories": [
+        "travel"
+    ],
+    "name": "Volkswagen AG",
+    "address": "Volkswagen AG\nPrivacy Team\nBerliner Ring 2\n38440 Wolfsburg\nGermany",
+    "email": "privacy@volkswagen.de",
+    "web": "https://www.volkswagenag.com/",
+    "sources": [
+        "https://datenschutz.volkswagen.de/"
```

Comment:
This page doesn't actually list the details in this record. Please specify the specific subpage(s).

Reply:
This page: https://datenschutz.volkswagen.de/impressum had mentioned all the details in the record, so I've updated it.